app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError

```

Comment:
According to the checklist, imports should be grouped with a blank line between different groups. The `datetime` import (standard library) should be separated from the local application imports (`app.errors`).